recipe/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "streamlit-avatar" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name.replace('-', '_') }}-{{ version }}.tar.gz"
+  sha256: 023893bd80db5a923d397fc64d27c7a972107da4f0bea0feb99a32f9363f2691
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  skip: True  # [py<37]
+
+requirements:
+  host:
+    - pip
+    - python
+    - wheel
+    - setuptools
+    - streamlit >=0.63
```

Comment:
Linter complaining about this pinning but I'm not sure why, isn't streamlit a pure python package?